.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Node.js CI
+
+on: [pull_request]
+
+jobs:
+  test-lambda-function:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./webhook
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+
+      - name: Install packages
+        # the IBI Group TSDX fork has some dependency issues
+        # that cause yarn install to fail on a ci runner. Disabling
+        # concurrency allows installation to complete successfully
+        run: yarn install --network-concurrency 1
+      - name: Lint code
+        run: yarn lint
+
+      - name: Semantic Release
+        if: github.event_name == 'push'
```

Comment:
If this job is only running on pull_request, will semantic-release run properly on the merge commit?

Reply:
Good catch! Fixed in bd0dd663a5578f8c305459c775f3a7881844d257